app/helpers/application_helper.rb

```diff
@@ -106,7 +107,9 @@ def link_to_resource(resource)
   end
 
   def manual_breadcrumb(items)
-    items.unshift ["Home", root_path]
+    home_text = "Home"
+    home_text = (global_locks.all?(&:warning?) ? warning_icon : lock_icon) + " " + home_text if global_locks.present?
```

Comment:
seems kinda overkill since we have the warning on every page anyway ?

Reply:
not _every_ page, only those with `render_locks` (which is usually the index pages)